### ISSUE TITLE
shared-lib-calls-exit 

### ISSUE THREAD

**Issue by luyatshimbalanga** [OpenImageDenoise/oidn#24]
I currently submit oidn 0.8.2 package for Fedora under [review](https://bugzilla.redhat.com/show_bug.cgi?id=1694553) and the reviewer reported this issue with that snippet:
```
shared-lib-calls-exit /usr/lib64/libOpenImageDenoise.so.0.8.2 exit@GLIBC_2.2.5
shared-lib-calls-exit:
This library package calls exit() or _exit(), probably in a non-fork()
context. Doing so from a library is strongly discouraged - when a library
function calls exit(), it prevents the calling program from handling the
error, reporting it to the user, closing files properly, and cleaning up any
state that the program has. It is preferred for the library to return an
actual error code and let the calling program decide how to handle the
situation.
```

**Comment by atafra** [comment]
Fixed in `v0.9.0`.

**Comment by luyatshimbalanga** [comment]
Thank you for the fix.
